oneBarangay/static/assets/js/volt.js

```diff
@@ -15,166 +15,166 @@
+      const slider = el.querySelector(':scope .input-slider')
+      const sliderId = slider.getAttribute('id')
+      const minValue = slider.getAttribute('data-range-value-min')
+      const maxValue = slider.getAttribute('data-range-value-max')
 
-      const sliderValue = el.querySelector(":scope .range-slider-value");
-      const sliderValueId = sliderValue.getAttribute("id");
-      const startValue = sliderValue.getAttribute("data-range-value-low");
+      const sliderValue = el.querySelector(':scope .range-slider-value')
+      const sliderValueId = sliderValue.getAttribute('id')
+      const startValue = sliderValue.getAttribute('data-range-value-low')
 
-      const c = d.getElementById(sliderId);
-      const id = d.getElementById(sliderValueId);
+      const c = d.getElementById(sliderId)
+      const id = d.getElementById(sliderValueId)
 
       noUiSlider.create(c, {
         start: [parseInt(startValue)],
         connect: [true, false],
         // step: 1000,
         range: {
           min: [parseInt(minValue)],
-          max: [parseInt(maxValue)],
-        },
-      });
-    });
+          max: [parseInt(maxValue)]
+        }
+      })
+    })
   }
 
-  if (d.getElementById("input-slider-range")) {
-    const c = d.getElementById("input-slider-range");
-    const low = d.getElementById("input-slider-range-value-low");
-    const e = d.getElementById("input-slider-range-value-high");
-    const f = [d, e];
+  if (d.getElementById('input-slider-range')) {
+    const c = d.getElementById('input-slider-range')
+    const low = d.getElementById('input-slider-range-value-low')
+    const e = d.getElementById('input-slider-range-value-high')
+    const f = [d, e]
 
     noUiSlider.create(c, {
-      start: [parseInt(low.getAttribute("data-range-value-low")), parseInt(e.getAttribute("data-range-value-high"))],
+      start: [parseInt(low.getAttribute('data-range-value-low')), parseInt(e.getAttribute('data-range-value-high'))],
       connect: !0,
       tooltips: true,
       range: {
-        min: parseInt(c.getAttribute("data-range-value-min")),
-        max: parseInt(c.getAttribute("data-range-value-max")),
-      },
+        min: parseInt(c.getAttribute('data-range-value-min')),
+        max: parseInt(c.getAttribute('data-range-value-max'))
+      }
     }),
-      c.noUiSlider.on("update", function (a, b) {
-        f[b].textContent = a[b];
-      });
+    c.noUiSlider.on('update', function (a, b) {
+      f[b].textContent = a[b]
+    })
   }
 
   // Chartist
 
-  if (d.querySelector(".ct-chart-sales-value")) {
+  if (d.querySelector('.ct-chart-sales-value')) {
     // Chart 5
     new Chartist.Line(
-      ".ct-chart-sales-value",
+      '.ct-chart-sales-value',
       {
-        labels: ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"],
-        series: [[0, 10, 30, 40, 80, 60, 100]],
+        labels: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
            labels: ["Mon", 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
```